certifikit-cli/src/main/kotlin/app/cash/certifikit/cli/http.kt

```diff
@@ -95,20 +103,20 @@ fun Main.fromHttps(host: String): List<X509Certificate> {
           val handshakeCertificates = HandshakeCertificates.Builder()
               .addTrustedCertificates(trustManager)
               .build()
-          sslSocketFactory(handshakeCertificates.sslSocketFactory(), handshakeCertificates.trustManager)
+          sslSocketFactory(handshakeCertificates.sslSocketFactory(),
+              handshakeCertificates.trustManager)
         }
       }
       .build()
 
-  val call = client.newCall(
-      Request.Builder()
-          .url("https://$host/")
-          .header("User-Agent", userAgent)
-          .build()
-  )
-
   val response = try {
-    call.execute()
+    client.newCall(
+        Request.Builder()
+            .url("https://$host/")
+            .header("User-Agent", userAgent)
+            .head()
+            .build())
+        .await()
```

Comment:
Main driver here is simple concurrency, to use for making other requests like OCSP (in other PRs).